openapi/document.yml

```diff
@@ -0,0 +1,145 @@
+    return a `baseUrl`. That `baseUrl` will become the base for the single document API described here. All API calls are scoped to the document ID
+    provided in the `auth` request path.
+
+
+paths:
+  /as-update:
+    get:
+      summary: Get As Yjs Update
+      description: |
+        Retrieves the document in Yjs update format, which provide a compressed binary representation of a set of changes to a Yjs document.
+
+        Updates can be applied to a Yjs document as follows:
+
+        ```javascript
+        import * as Y from 'yjs'
+
+        let doc = new Y.Doc()
+        doc.transact(() => {
+          Y.applyUpdate(doc, update)
+        })
+        ```
+      security:
+        - bearerAuth: []
+      responses:
+        '200':
+          description: Document update
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '404':
+          description: Document not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /update:
+    post:
+      summary: Update With Yjs Update
+      description: |
+        Writes to a document with the provided Yjs update, which provide a compressed binary representation of a set of changes to a Yjs document.
+
+        Updates can be obtained from a Yjs document as follows:
+
+        ```javascript
+        import * as Y from 'yjs'
+
+        // doc is a Y.Doc
+        let update = Y.encodeStateAsUpdate(doc)
+        ```
+
+        Note that this will encode the *entire* document state as an update, but you can also use this endpoint for incremental updates.
+        See Yjs's [Document Updates](https://docs.yjs.dev/api/document-updates) documentation for more information.
+      security:
+        - bearerAuth: []
+      requestBody:
+        required: true
+        content:
+          application/octet-stream:
+            schema:
+              type: string
+              format: binary
+      responses:
+        '200':
+          description: Document updated
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '404':
+          description: Document not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+  /ws/{docId}:
```

Comment:
This is introduced in `v0.5.1` (the next version), right? If that's the case, maybe we should update the version in these docs to `v0.5.1` (and publish `v0.5.1` of the y-sweet server) before we release the docs?